docs/env-migration-matrix.md

```
@@ -0,0 +1,320 @@
+| Key | Status | Action |
+|-----|--------|--------|
+| `AXON_WEB_API_TOKEN` | ✅ Added to runtime.rs registry as KeepEnv (ztqd.1) | Done |
+| `CHROME_URL` | ✅ Added to migration.rs as Delete (ztqd.1) | Done |
+| `TEI_MAX_BATCH_TOKENS` | ✅ Added to advanced.rs as ComposeEnv (ztqd.1) | Done |
+
+---
+
+## TOML [services] Section — Action Required
+
+`src/core/config/parse/toml_config.rs` still has:
+```rust
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)